client/src/components/relatedProducts/outfitList.jsx

```diff
@@ -18,48 +18,81 @@ const responsive = {
 };
 
 const Outfit = (props) => {
-  const [productData, setProductData] = React.useState([]);
-  const [list, setList] = React.useState([<AddOutfit key = {0} handleClick = {()=>{console.log('test add button')}}/>])
+  const [productData, setProductData] = React.useState(null);
```

Comment:
Just FYI, you can deconstruct the useState from react in the import statement like this:
`import React, {useState} from 'react';`